bpf/cilium-map-migrate.c

```diff
@@ -36,6 +36,7 @@
 #include "elf/libelf.h"
 #include "elf/gelf.h"
 
+#include "bpf/ctx/skb.h"
```

Comment:
To otherwise prevent a compile error since api.h either pulls in skb or xdp helpers. But I'll change it to unspec.h since we don't care about the context there.
```
In file included from include/bpf/helpers.h:9:0,
                 from include/bpf/api.h:14,
                 from cilium-map-migrate.c:40:
include/bpf/ctx/ctx.h:8:3: error: #error "No __ctx_buff context defined. Please either include 'bpf/ctx/skb.h' or 'bpf/ctx/xdp.h'."
 # error "No __ctx_buff context defined. Please either include 'bpf/ctx/skb.h' or 'bpf/ctx/xdp.h'."
   ^~~~~
```